rtgui/editorpanel.cc

```diff
@@ -46,6 +46,8 @@ class EditorPanel::MonitorProfileSelector
     void prepareProfileBox ()
     {
         profileBox.set_size_request (100, -1);
+        profileBox.setPreferredWidth(90, 150);
```

Comment:
As I don't really care of that, you can open an issue for adding this to rawtherapee.astylerec, but it will touch really a lot of files and should be applied in a "critical ~~section~~ moment".